setup.py

```diff
@@ -5,11 +5,12 @@
 import sys
 import setuptools
 import subprocess
+from warnings import warn
 
 
 ERFAPKGDIR = os.path.abspath(os.path.dirname(__file__))
-
-ERFA_SRC = os.path.abspath(os.path.join(ERFAPKGDIR, 'liberfa', 'erfa', 'src'))
+LIBERFADIR = os.path.abspath(os.path.join(ERFAPKGDIR, 'liberfa', 'erfa'))
+ERFA_SRC = os.path.abspath(os.path.join(LIBERFADIR, 'src'))
```

Comment:
Please consider to adjust the above lines to avoid absolute paths (see also #29).

Reply:
Done.

Reply:
Please also:

```ERFA_SRC = os.path.abspath(os.path.join(LIBERFADIR, 'src'))```

should become

```ERFA_SRC = os.path.join(LIBERFADIR, 'src')```

Also the `Extension` object seems to injects paths into the egg-info/SOURCES.txt